snscrape/modules/twitter.py

```diff
@@ -359,6 +368,25 @@ def _tweet_to_tweet(self, tweet, obj):
 			User(username = u['screen_name'], displayname = u['name'], id = u['id'] if 'id' in u else int(u['id_str'])) \
 			for u in tweet['entities']['user_mentions']
 		  ] if 'user_mentions' in tweet['entities'] and tweet['entities']['user_mentions'] else None
+		
+		# https://developer.twitter.com/en/docs/tutorials/filtering-tweets-by-location								     
```

Comment:
Oh, I missed all the trailing spaces on various lines from here on previously as they aren't shown on the web interface.

Reply:
There's still a bunch of trailing tab characters below here and above on `kwargs['source']`.

Reply:
Opened the file with vim and fixed them :)